### PR TITLE
LPS-102767, site settings ui reference rewrite

### PR DESCRIPTION
**Pull request by JamesAGarcia**
Here is the first article from LPS-102767.

### CODE REVIEW

**Comment by JamesAGarcia** [comment]
I will update this PR with the second article edit for LPS-102767 when it's ready--I forgot all about adding the new article to relevant RSTs and creating a landing card for it.

File: docs/dxp/7.x/en/site-building/creating-pages/page-settings/configuring-individual-pages.md [hunk 1]
```diff
@@ -1,139 +1,233 @@
...
 
-1. Optionally click the *Change Current Theme* button to choose a different Theme.
+## SEO
 
-1. Modify the available theme settings, color scheme(s), and CSS for the Theme and click *Save* to apply the changes to the page. See the [Configuring Page Sets](./05-configuring-page-sets.md#current-theme) for more information on the available settings for themes.
+Search engine optimization (SEO) refers to the methods used to improve your page's ranking in search engine results pages (SERP). The *SEO* tab provides easy access to settings you can use to optimize your page's content. See [Search Engine Optimization]() for more information about the following settings. <!--Rename current placeholder, "SEO Optimization," to "Search Engine Optimization".-->
```
Comment (JamesAGarcia) [inline docs/dxp/7.x/en/site-building/creating-pages/page-settings/configuring-individual-pages.md]: The current placeholder is just in an RST file. I should have said that I'm waiting for the Lima article to update the link path.

**Comment by JamesAGarcia** [comment]
The final image (commit e254b73) is simply cosmetic; the Lima image is better than the image I initially used.

**Comment by Alec-Shay** [comment]
Forgot to close this previously.

See: https://github.com/sez11a/liferay-learn/pull/125

**Pull request closed without merging**